ansible_collections/arista/avd/plugins/plugin_utils/utils/python_to_ansible_logging_handler.py

```diff
@@ -44,6 +44,17 @@ def emit(self, record: LogRecord) -> None:
             self.result.setdefault("warnings", []).append(message)
         elif record.levelno == logging.INFO:
             self.display.v(str(message))
+        # logging.INFO is 20, logging.DEBUG is 10. There is some room
+        elif record.levelno == logging.INFO - 1:
+            self.display.vv(str(message))
+        elif record.levelno == logging.INFO - 2:
+            self.display.vvv(str(message))
+        elif record.levelno == logging.INFO - 3:
+            self.display.vvvv(str(message))
+        elif record.levelno == logging.INFO - 4:
+            self.display.vvvvv(str(message))
+        elif record.levelno == logging.INFO - 5:
+            self.display.vvvvvv(str(message))
```

Comment:
These changes can go - I was experimenting a bit with log level and we have room between debug and info to implement some verbosity if we want. Reviewer?

Reply:
I don't think we should play with all this. Debug (-vvv) and info (-v) should be enough.

Reply:
ok in that case I can do it in the PR (I had remove it in one of my last comments)